backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -230,10 +276,16 @@ public long getPatientsCount(
     if (namePrefixMatch != null && namePrefixMatch.trim().length() < MINIMUM_CHAR_FOR_SEARCH) {
       return 0;
     }
-    return _repo.count(
-        buildPersonSearchFilter(
-            facilityId, isArchived, namePrefixMatch, includeArchivedFacilities));
+
+    var filter =
+        isArchived
```

Comment:
nit: in my mind the parameter here makes more sense to be `includeArchived` since we're optionally including count of archived patients, and then when we use `isArchived` it should be only getting patients that are archived